### ISSUE TITLE
RSVP list visible to regular users

### ISSUE THREAD

**Issue by ballek** [drenfr01/unionCapital#163]
When I click on the More Info Link as a regular user, I can see those who have RSVP'd to this event. I think that for privacy concerns we had said that this list should only be visible to the organizers of the event?

![img_7590](https://cloud.githubusercontent.com/assets/7898004/7776662/84b4e902-008a-11e5-803f-de02f1d1af59.JPG)

**Comment by drenfr01** [comment]
@ballek So I tried to replicate this on Modulus QA, but couldn't. Are you sure you weren't logged in as AA when you looked at the RSVP list? Because the RSVP list is designed to show you your own RSVP...

Here's  how I tried to replicate the bug:

1) logged in as test25@gmail.com
2) RSVP'd to Test Safari event with 2 users
3) created new user test26@gmail.com
4) logged in to rsvp
5) saw below screenshot, where it appeared to function correctly.
